assets/app/data/crowbar/services/crowbar-backup.factory.spec.js

```diff
@@ -36,7 +36,7 @@ describe('Crowbar Backup Factory', function () {
         describe('when create method is executed', function () {
 
             beforeEach(function () {
-                $httpBackend.expect('POST', '/utils/backups', undefined)
+                $httpBackend.expect('POST', '/api/upgrade/adminbackup', undefined)
```

Comment:
Please, test Headers V2 are being sent as well
@see https://docs.angularjs.org/api/ngMock/service/$httpBackend
`expect(method, url, [data], [headers], [keys]);` section for method definition, and `Unit testing with mock $httpBackend` for an example.